app/pages/homepage/ui.test.js

```diff
@@ -41,6 +53,29 @@ test('should navigate to home page header banner', async (t) => {
     .expect(getLocation()).eql('http://localhost:1234/');
 });
 
+test('should display the login link and navigate to the login page when clicking the login button', async (t) => {
+  await pageSetup(t);
+
+  nock('http://identity-server')
+    .get('/login')
+    .reply(200);
+
+  const getLocation = ClientFunction(() => document.location.href);
+  const loginComponent = Selector('[data-test-id="login-logout-component"] a');
+  await t
+    .expect(await extractInnerText(loginComponent)).eql('Log in')
+    .click(loginComponent)
+    .expect(getLocation()).eql('http://identity-server/login');
+});
+
+test('should display the logout link when the user is authenticated', async (t) => {
```

Comment:
add test for what is displayed when not logged in?

Reply:
The test above this one covers this. I could be more explicit in the name of that test and say when not authenticated